.github/workflows/lint.yml

```
@@ -50,3 +50,19 @@ jobs:
 
       - name: Run linter
         uses: ludeeus/action-shellcheck@2.0.0
+        with:
+          # Names of scripts with custom shebangs that the action would not
+          # discover on its own:
+          additional_files: >-
+            check_certificate_renewal
+            cpcd-config-up
+            discovery
+            finish
+            lock-tables-for-backup
+            matter-server-discovery
+            reboot
+            run
+            shutdown
+            universal-silabs-flasher-up
+            unlock-tables-for-backup
+            vlc-discovery
```

It seems that the Google Assistant SDK add-on doesn't build anymore. Let's exclude it from linting so we get a complete CI run.

```suggestion
            vlc-discovery
          ignore_paths: >-
            google_assistant
```